.github/workflows/perf.yml

```diff
@@ -0,0 +1,85 @@
+name: libp2p perf test
+
+# How to configure a repository for running this workflow:
+# 1. Run 'make ssh-keygen' in 'perf' to generate a new SSH key pair named 'user' in 'perf/terraform/region/files'
+# 2. Configure auth for the AWS provider as per https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication-and-configuration
+# 3. Run 'terraform init' and 'terraform apply' in 'perf/terraform/remote' to create the long-lived AWS resources
+# 4. Go to https://console.aws.amazon.com/iamv2/home?#/users/details/perf?section=security_credentials
+# 5. Click 'Create access key' to get the access key ID and secret access key
+# 6. Go to https://github.com/libp2p/test-plans/settings/secrets/actions
+# 7. Click 'New repository secret', set the name to 'PERF_AWS_SECRET_ACCESS_KEY', and paste the secret access key from step 5
+# 8. Click 'New repository secret', set the name to 'PERF_SSH_PRIVATE_KEY', and paste the private key from step 1
+# 9. Go to https://github.com/libp2p/test-plans/settings/variables/actions
+# 10. Click 'New repository variable', set the name to 'PERF_AWS_ACCESS_KEY_ID', and paste the access key ID from step 5
+
+on:
+  workflow_dispatch:
```

Comment:
> > You'll be able to trigger the workflow only after it is merged to the default branch.

> It is necessary to trigger the workflow before merging into the default branch (i.e. master). Otherwise how would one test a release candidate. Are there any larger blockers to triggering it before the merge?

Sorry, I didn't make it clear enough. I only meant that before we can trigger the workflow for the **first** time, these changes will have to make it to master. Once it does, we'll be able to trigger the workflow from PR branches too.

Reply:
Thanks. That makes sense.